app/assets/javascripts/cases.js

```diff
@@ -1,6 +1,6 @@
-document.addEventListener("DOMContentLoaded", function() {
-	const button = document.getElementById('avatar_browse');
-	button.onclick = function(){
-		$('input[id=lefile_article]').click();
-	}
-});
\ No newline at end of file
+// document.addEventListener("DOMContentLoaded", function() {

```

Comment:
Did you mean to include this as part of the commit?

Reply:
I didn't. I should address those event handlers separately.

Reply:
This has been done,